bearuploader/Constants.py

```diff
@@ -0,0 +1,72 @@
+# Constants to be used for the uploadTool to improve automation
+
+classifiers = """[
+              'Development Status :: 4 - Beta',
+
+              'Environment :: Console',
+              'Environment :: MacOS X',
+              'Environment :: Win32 (MS Windows)',
+              'Environment :: X11 Applications :: Gnome',
+
+              'Intended Audience :: Science/Research',
+              'Intended Audience :: Developers',
+
+              'License :: OSI Approved :: GNU Affero General Public License '
+              'v3 or later (AGPLv3+)',
+
+              'Operating System :: OS Independent',
+
+              'Programming Language :: Python :: Implementation :: CPython',
+              'Programming Language :: Python :: 3.3',
+              'Programming Language :: Python :: 3.4',
+              'Programming Language :: Python :: 3.5',
+              'Programming Language :: Python :: 3 :: Only',
+
+              'Topic :: Scientific/Engineering :: Information Analysis',
+              'Topic :: Software Development :: Quality Assurance',
+              'Topic :: Text Processing :: Linguistic']"""
+imports="""# Start ignoring PyImportSortBear as imports below may yield syntax errors
+from bears import assert_supported_version
+assert_supported_version()
+# Stop ignoring
+
+import locale
+import sys
+from subprocess import call
+
+import setuptools.command.build_py
+from bears import Constants
+from setuptools import find_packages, setup
+from setuptools.command.test import test as TestCommand"""
+
+encoding="""try:
+    locale.getlocale()
+except (ValueError, UnicodeError):
+    locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')"""
+
+requirements="""with open('requirements.txt') as requirements:
+    required = requirements.read().splitlines()
+
+with open('test-requirements.txt') as requirements:
+    test_required = requirements.read().splitlines()
+
+with open("README.rst") as readme:
+    long_description = readme.read()"""
+
+setup_constants="""author="The coala developers",
+          maintainer="Lasse Schuirmann, Fabian Neuschmidt, Mischa Kr\xfcger",
+          maintainer_email=('lasse.schuirmann@gmail.com, '
+                            'fabian@neuschmidt.de, '
+                            'makman@alice.de'),
+          url='http://coala.rtfd.org/',
+          platforms='any',
+          packages=find_packages(exclude=("build.*", "tests", "tests.*")),
+          install_requires=required,
+          tests_require=test_required,
+          package_data={'bears': ["VERSION"],
+                        'bears.java': ['checkstyle.jar', 'google_checks.xml'],
```

Comment:
Having all this bear specific data disappear is going to be a TODO. In principle the whole setup.py should be tailored exactly to what an individual bear needs, not more, not less.